main.js

```diff
@@ -0,0 +1,129 @@
+        areaP.textContent = `Area: ${area}`;
+        h2Area.textContent = city;
+    } else {
+        areaP.textContent = `Nearest Area: ${area}`;
+        h2Area.textContent = city;
+    }
+    current.append(areaP);
+
+    let region = weatherData.nearest_area[0].region[0].value;
+    let regionP = document.createElement('p');
+    regionP.textContent = `Region: ${region}`;
+    current.append(regionP);
+
+    let country = weatherData.nearest_area[0].country[0].value;
+    let countryP = document.createElement('p');
+    countryP.textContent = `Country: ${country}`;
+    current.append(countryP);
+
+    let currently = weatherData.current_condition[0].FeelsLikeF;
+    let currentlyP = document.createElement('p');
+    currentlyP.textContent = `Feels like it's ${currently}°F.`;
+    current.append(currentlyP);
+
+    const chanceOfSunshine = weatherData.weather[0].hourly[0].chanceofsunshine;
+    const sunny = document.createElement('p')
+    sunny.textContent = `Chance of Sunshine: ${chanceOfSunshine} %`
+    current.append(sunny);
+
+    const chanceOfRain = weatherData.weather[0].hourly[0].chanceofrain;
+    const rainy = document.createElement('p')
+    rainy.textContent = `Chance of Rain: ${chanceOfRain} %`
+    current.append(rainy);
+
+    const chanceOfSnow = weatherData.weather[0].hourly[0].chanceofsnow;
+    const snow = document.createElement('p')
+    snow.textContent = `Chance of Snow: ${chanceOfSnow} %`
+    current.append(snow);
+
+    for (let i = 0; i < weatherData.weather[0].hourly.length; i++) {
+        if (Number(weatherData.weather[0].hourly[i].chanceofsunshine) > 50) {
+            weatherIcon.src = "./assets/icons8-summer.gif";
+            weatherIcon.alt = "sun"
+        }
+        if (Number(weatherData.weather[0].hourly[i].chanceofrain) > 50) {
+            weatherIcon.src = "./assets/icons8-torrential-rain.gif"
+            weatherIcon.alt = "rain"
+        }
+        if (Number(weatherData.weather[0].hourly[i].chanceofsnow) > 50) {
+            weatherIcon.src = "./assets/icons8-light-snow.gif"
+            weatherIcon.alt = "snow"
+        }
+    }
+    current.prepend(weatherIcon);
+}
+
+//
+//
+//
+let widget = document.querySelector('#widget');
+
+widget.addEventListener('submit', (event) => {
+    event.preventDefault();
+    let temp = Number(document.querySelector('#temp-to-convert').value);
+    let finalResult = document.querySelector('#result');
+    const celsius = document.querySelector('#to-c');
+    const fahrenheit = document.querySelector('#to-f')
+    //https://stackoverflow.com/questions/9618504/how-to-get-the-selected-radio-button-s-value
+
+    let result;
+    if (celsius.checked) {
+      result = (temp - 32) * (5/9)
+      finalResult.textContent = `${temp}°F = ${result.toFixed(2)}°C`;
+    } else if (fahrenheit.checked) {
+      result = (temp * (9/5)) + 32
+      finalResult.textContent = `${temp}°C = ${result.toFixed(2)}°F`;
+    }  
+}
+);
+
+//
+//
+// 
+
```

Comment:
At the moment, every time a search is made the new location is added to the page instead of replacing the previous search. 

There are a few ways to handle this. You can have a conditional that checks whether that code was already created an needs to be updated or if it wasn't create it - you create it. 

Or you can delete the whole section and create it every time a search is made. 

Those are just two ways to do it. I am sure there are more.